packages/core/package.json

```diff
@@ -40,8 +40,7 @@
     "@types/json-schema": "^7.0.15"
   },
   "devDependencies": {
-    "json-schema": "^0.4.0",
```

Comment:
Just double-checking if this removal is intentional?

Reply:
Yes, it's because we actually don't depend on `json-schema` to build this package. Only the dependency on `@types/json-schema` is used in production.